app/controllers/curation_concerns/curation_concerns_controller.rb

```diff
@@ -1,3 +1,4 @@
+# rubocop:disable Metrics/ClassLength
```

Comment:
Want to make a ticket to refactor this controller?  I don't think it's a big deal if the controller is over 100 lines, but it would be good to break out the functionality into logical chunks as it continues to grow.